sdk/utils.ts

```diff
@@ -173,7 +177,7 @@ export const getAlerts = async (query: AlertQueryOptions): Promise<AlertsRespons
   return response.data.data.alerts
 }
 
-export const fetchJwt = async (claims: {}, expiresAt?: Date): Promise<{token: string} | null> => {
+export const fetchJwt = async (claims: {}, axiosInstance: AxiosInstance = getAxiosInstance() ,expiresAt?: Date): Promise<{token: string} | null> => {
```

Comment:
this would change the signature of this function and potentially break existing usage. if anything, the `axiosInstance` param should be the third argument. if we put it second because the TS compiler was complaining about default args, we can set the `expiresAt` default value to undefined and leave it as the second param

Reply:
it would be better though to use a `provideFetchJwt` method to return the `fetchJwt` method. this would allow for injecting mocks easily without affecting the method signature

Reply:
Added a provider method for this.